### ISSUE TITLE
[2023-09-07] Ingestion Request: 16th International Natural Language Generation Conference

### ISSUE THREAD

**Issue by ChunJiChen** [acl-org/acl-anthology#2730]
### General information about this request

- [X] I confirm that I have read the [Information for Submitters](https://aclanthology.org/info/contrib/).
- [ ] I am submitting a request for a **new venue** that does not exist in the ACL Anthology yet.

### Venue Identifier

inlg

### Volume Title

Proceedings of the 16th International Conference on Natural Language Generation

### Date of Publication

2023-09-05

**Comment by anthology-assist** [comment]
@ChunJiChen could you follow Submit your data section [here](https://aclanthology.org/info/contrib/) to submit ingestion material?

**Comment by ChunJiChen** [comment]
@anthology-assist Sorry for a bit delay. I already sent the date to the Anthology Director. Please help us to upload it by Sep. 7th, and let me know if there are any problems. Thank you very much!

**Comment by ChunJiChen** [comment]
@anthology-assist Thank you for the information!

There are three volumes in INLG-2023, and here is the data generated by the Softconf system: https://drive.google.com/file/d/1ILi8yAVW4FaN9RsN-DNp_T99LRLuiZAy/view?usp=sharing

Some workshops also want to publish the proceedings on ACL Anthology. However, the workshops use different platforms for submissions. They follow the aclpub2 format (https://github.com/rycolab/aclpub2) to prepare the workshop proceedings. It is different from the format of the main proceedings, and thus I did not merge the data together. Below are the data for these workshops. Please let me know if there are any problems, I will correct them or contact workshop organizers to correct them.

(1) DSTC11: The Eleventh Dialog System Technology Challenge;
DSTC11: https://drive.google.com/file/d/1PMVMxAumWj1dPo54xtMajKDNrsDHPtJd/view?usp=sharing

(2) The 1st Workshop on Counter Speech for Online Abuse
CS4OA: https://drive.google.com/drive/folders/1CuJHsj4sZP5W1xdatt7uWar24ixSAELA?usp=drive_link

(3) Connecting multiple disciplines to AI techniques in interaction-centric autism research and diagnosis
ICARD: https://drive.google.com/file/d/1m420mCs0JlkxNRrpD8mWi9muz-OhpNJY/view?usp=sharing

(4) Workshop on Multimodal, Multilingual Natural Language Generation and Multilingual WebNLG Challenge
MM-NLG2023: https://drive.google.com/file/d/1w7titpmrFajvmgAn94wD75Ael8opSRLR/view

Again, thank you very much.

**Comment by anthology-assist** [comment]
@ChunJiChen 

MM-NLG2023 needs a new link:
<img width="1137" alt="Screenshot 2023-09-01 at 3 28 24 PM" src="https://github.com/acl-org/acl-anthology/assets/126604033/21df8e04-0479-4d3c-acc0-b7229d6ed5b5">

CS4OA doesn't follow acl_pub2 format. Please submit the `output` folder (instead of just the `input`). See: https://github.com/rycolab/aclpub2

All other volumes have been ingested, please preview here #2764

**Comment by ChunJiChen** [comment]
@anthology-assist Thank you so much!

Here is the new link for MM-NLG2023: https://drive.google.com/file/d/1V8EnAmjTQ1Nvyvad9vETaOZPxd-VlQGB/view?usp=sharing

I already informed CS4OA organizers to provide the "output" folder. I'll inform you after they update it.

**Comment by ChunJiChen** [comment]
@anthology-assist Here is the output folder of CS4OA
https://drive.google.com/drive/folders/1CuJHsj4sZP5W1xdatt7uWar24ixSAELA?usp=sharing

**Comment by anthology-assist** [comment]
@ChunJiChen cs4oa is ingested. MM-NLG ingestion material is incorrect.

**Comment by albertgatt** [comment]
Hi. I've replaced the MM-NLG ingestion material. Here is the link again: [https://drive.google.com/file/d/13aCkS8Z2E7wJcb8F2iZNNAAfICJhSJEX/view?usp=sharing](https://drive.google.com/file/d/13aCkS8Z2E7wJcb8F2iZNNAAfICJhSJEX/view?usp=sharing)

 Could you please check again and let me know if it's ok?

**Comment by ChunJiChen** [comment]
@anthology-assist Thank you very much for helping us to prepare the proceedings.

We noticed one error from our side, after I shared the preview version with other organizers.

The editor of "2023.inlg-genchal" (https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-genchal/) should be "Simon Mille" instead of "Maria Keet, Hung-Yi Lee, Sina Zarrieß."

Thus, there are two pages that need to be corrected: 
(1) https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-genchal/
(2) https://preview.aclanthology.org/inlg-23-ingestion/2023.inlg-genchal.0/

I prepared the revised "meta" and "2023.inlg-3.0-bib" in case you need. (https://drive.google.com/drive/folders/1ObYWuIzI_muQeqjCpk7WW1n4io46p3Dm?usp=sharing)

Please let me know if we need to prepare other materials for it. 

Apologize for the inconvenience caused.

**Comment by anthology-assist** [comment]
@ChunJiChen All volumes are ingested and the editor issue is fixed.

**Comment by ChunJiChen** [comment]
@anthology-assist Thank you so much!

**Comment by ChunJiChen** [comment]
Hi @anthology-assist 

I noticed that the appendix is unavailable on the preview page (https://preview.aclanthology.org/inlg-23-ingestion/2023.inlg-1.29/)

Will it become available when the final version comes out? 

Again, thank you so much for your help.

**Comment by ChunJiChen** [comment]
Hi @anthology-assist 

I noticed that one of the editor names is not correct...

For the following two proceedings, "Maria Keet" should be replaced by "C. Maria Keet" (https://aclanthology.org/people/c/c-maria-keet/)

https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-1/

https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-demos/

Could you help me to correct it, please?

**Comment by anthology-assist** [comment]
> Hi @anthology-assist
> 
> I noticed that the appendix is unavailable on the preview page (https://preview.aclanthology.org/inlg-23-ingestion/2023.inlg-1.29/)
> 
> Will it become available when the final version comes out?
> 
> Again, thank you so much for your help.

@ChunJiChen Appendix is unavailable because it's not included in the ingestion material.

**Comment by anthology-assist** [comment]
> https://aclanthology.org/people/c/c-maria-keet/

@ChunJiChen Corrected.

**Comment by anthology-assist** [comment]
> Hi @anthology-assist
> 
> I noticed that one of the editor names needs to be corrected on the webpage. I think it is a problem from my side, and I'm sorry about it.
> 
> For the following two proceedings, "Maria Keet" should be replaced by "C. Maria Keet" (https://aclanthology.org/people/c/c-maria-keet/)
> 
> https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-1/
> 
> https://preview.aclanthology.org/inlg-23-ingestion/volumes/2023.inlg-demos/
> 
> Could you help me to correct it, please?

Corrected @ChunJiChen

**Comment by ChunJiChen** [comment]
> > Hi @anthology-assist
> > I noticed that the appendix is unavailable on the preview page (https://preview.aclanthology.org/inlg-23-ingestion/2023.inlg-1.29/)
> > Will it become available when the final version comes out?
> > Again, thank you so much for your help.
> 
> @ChunJiChen Appendix is unavailable because it's not included in the ingestion material.

Hi @anthology-assist , 

Thank you for the information. 

I checked the data I shared (
https://drive.google.com/file/d/1ILi8yAVW4FaN9RsN-DNp_T99LRLuiZAy/view?usp=sharing)

I think the data for the appendix are in the path: "2023.inlg-main/proceedings/cdrom/additional". Because I generated the data by using the SoftConf system, I did not modify any part of the generated data. I assumed the data in the appendix would be on the right path. 

I'm sorry that I may miss some rules. Could you let me know what is the expected way to provide appendix data, please?


Many thanks,
Chung-Chi

**Comment by anthology-assist** [comment]
@ChunJiChen Apologies. The additional material is indeed there and the missing appendix issue is fixed after the reingestion of the main volume (e.g. https://aclanthology.org/attachments/2023.inlg-main.29.Supplementary_Attachment.pdf)

**Comment by ChunJiChen** [comment]
Thank you so much!

**Comment by ChunJiChen** [comment]
Hi @anthology-assist 

May I know the expected time that our proceedings will be officially available on ACL Anthology?

**Comment by evanmiltenburg** [comment]
Asking again: @anthology-assist could you tell us whether anything is needed for ingestion? If everything is in place, can the proceedings please be ingested?

**Comment by mjpost** [comment]
Sorry everyone, this was my fault. I didn't realize my comments had been addressed. I just merged it, and it should be live within an hour.

**Comment by evanmiltenburg** [comment]
Thanks! Perhaps a separate issue, but I'll ask here to be sure: INLG and SIGDIAL were collocated this year. I.e. they were held in parallel at the same venue. But they are like siblings; they are not sub-events of one another. Could/should this be recorded in the Anthology? (For the workshops, we have asked organisers how they would like their venue to be recorded, so they indicated whether they prefer SIGDIAL or INLG as the parent venue --not ideal but I guess it works.)

**Comment by nschneid** [comment]
+ can you please add the SIGs for the respective events (INLG: GEN, SIGDIAL: DIAL)

**Comment by mjpost** [comment]
Okay. @anthology-assist can you please, I guess in a separate PR:

* Add a `<venue>sigdial</venue>` tag to all INLG 2023 main conference volumes
* Add a `<venue>ws</venue>` tag to all workshops that were just imported
* Add the SIG affiliations per @nschneid's request above?

We could also create an event block in the `2023.sigdial.xml` file and list all these workshops as affiliated with it, too, to they'll appear on both event pages (https://aclanthology.org/events/sigdial-2023/ and https://aclanthology.org/events/inlg-2023/).
